apply-notch-hax.sh

```diff
@@ -42,7 +60,7 @@ if ! patch -p1 < $DIFF; then
           Please adjust '$DEVICE.diff' and try again!"
 	exit 1
 fi
-cd ../
+cd ..
```

Comment:
Could you add the `/` back here? I suppose it got lost in the `popd` manual reversal